app/src/main/java/com/vero/tasky/agenda/data/remote/network/dto/PhotoDto.kt

```diff
@@ -0,0 +1,9 @@
+package com.vero.tasky.agenda.data.remote.network.dto
+
+import com.squareup.moshi.JsonClass
+
+@JsonClass(generateAdapter = true)
+data class PhotoDto (
+    val key: String,
+    val url: String,
```

Comment:
When a photo is added to the event (before saving the event), the photo only has a uri (not key nor url). So you'd need to somehow differentiate between Local and Remote photos